specification/Metadata/Semantics/README.md

```diff
@@ -0,0 +1,190 @@
+
+`TILE_*` semantics provide meaning for properties associated with a particular tile, and should take precedence over equivalent metadata on parent objects, as well as over values derived from subdivision schemes like [3DTILES_implicit_tiling](../../../extensions/3DTILES_implicit_tiling).
+
+In particular, `TILE_BOUNDING_BOX`, `TILE_BOUNDING_REGION`, and `TILE_BOUNDING_SPHERE` semantics each define a tighter bounding volume for a tile than is implicitly calculated from [3DTILES_implicit_tiling](../../../extensions/3DTILES_implicit_tiling). If more than one of these semantics are available for a tile, clients may select the most appropriate option based on use case and performance requirements.
+
+#### `TILE_BOUNDING_BOX`
+
+The bounding volume of the tile, expressed as a [box (as defined by 3D Tiles 1.0)](../../../specification#box). Equivalent to `tile.boundingVolume.box`.
+
+* Type: `ARRAY`
+* Component type: `FLOAT32` or `FLOAT64`
+* Component count: `12`
+
+#### `TILE_BOUNDING_REGION`
+
+The bounding volume of the tile, expressed as a [region (as defined by 3D Tiles 1.0)](../../../specification#region). Equivalent to `tile.boundingVolume.region`.
+
+* Type: `ARRAY`
+* Component type: `FLOAT64`
+* Component count: `6`
+
+#### `TILE_BOUNDING_SPHERE`
+
+The bounding volume of the tile, expressed as a [sphere (as defined by 3D Tiles 1.0)](../../../specification#sphere). Equivalent to `tile.boundingVolume.sphere`.
+
+* Type: `ARRAY`
+* Component type: `FLOAT32` or `FLOAT64`
+* Component count: `4`
+
+#### `TILE_MINIMUM_HEIGHT`
+
+The minimum height of the tile above (or below) the WGS84 ellipsoid. Equivalent to minimum height component of `TILE_BOUNDING_REGION` and `tile.boundingVolume.region`.
+
+* Type: `FLOAT32` or `FLOAT64`
+
+#### `TILE_MAXIMUM_HEIGHT`
+
+The maximum height of the tile above (or below) the WGS84 ellipsoid. Equivalent to maximum height component of `TILE_BOUNDING_REGION` and `tile.boundingVolume.region`.
+
+* Type: `FLOAT32` or `FLOAT64`
+
+#### `TILE_HORIZON_OCCLUSION_POINT`
+
+The horizon occlusion point of the tile expressed in an ellipsoid-scaled fixed frame. If this point is below the horizon, the entire tile is below the horizon. See [Horizon Culling](https://cesium.com/blog/2013/04/25/horizon-culling/) for more information.
+
+* Type: `VEC3`
+* Component type: `FLOAT32` or `FLOAT64`
+
+> **Implementation Note**: `TILE_HORIZON_OCCLUSION_POINT` should account for all content in a tile and its descendants, whereas `CONTENT_HORIZON_OCCLUSION_POINT` should only account for content in a tile. When the two values are equivalent, only `TILE_HORIZON_OCCLUSION_POINT` should be specified.
+
+#### `TILE_GEOMETRIC_ERROR`
+
+The geometric error of the tile. Equivalent to `tile.geometricError`.
+
+* Type: `FLOAT32` or `FLOAT64`
+
+### Content
+
+#### Overview
+
+`CONTENT_*` semantics provide meaning for properties associated with a particular tile content, and should take precedence over equivalent metadata on the containing tile.
```

Comment:
> and should take precedence over equivalent metadata on the containing tile

I see them more as complementary than taking precedence over the other. The tile's bounding volume is used for spatial coherence while the content bounding volume is used for finer grained culling, both serving different purposes.

I guess the main point is that `3DTILES_implicit_tiling` doesn't implicitly define content bounding volumes so this is how to define them.

Reply:
We should probably mention that if `3DTILES_multiple_contents` is used this defines the content bounding volume for all the contents in the tile.